docs/ReleaseNotes_12.0.md

```diff
@@ -0,0 +1,89 @@
+- Created basic layer movement functionality (up, down) by @affirVega in https://github.com/flameshot-org/flameshot/pull/2108
+<p align=center><img src="https://github.com/flameshot-org/flameshot/blob/master/docs/images/layer.png" width=50%> </p>
+
+- Added a new widget to allow the colorwheel to be more easily customized by https://github.com/flameshot-org/flameshot/pull/2202
+<p align=center><img src="https://github.com/flameshot-org/flameshot/blob/master/docs/images/colorwheel.png" width=50%> </p>
+
+
+- Added magnifier for more precise selections by @SilasDo in https://github.com/flameshot-org/flameshot/pull/2219
+  - The new magnifier can be enabled in ```Configuration > General > Show Magnifier``` 
+  - There is an option to make the magnifier a square or circle
+<p align=center><img src="https://github.com/flameshot-org/flameshot/blob/master/docs/images/magnfier.png" width=50%> </p>
+
+- Incremental markers can now have a point if you drag when placing them. @vozdeckyl in https://github.com/flameshot-org/flameshot/pull/2638
+<p align=center><img src="https://github.com/flameshot-org/flameshot/blob/master/docs/images/number_pointer.png" width=50%> </p>
+
+- Added the ability to cache the last region by @borgmanJeremy in https://github.com/flameshot-org/flameshot/pull/2615
+  - The launcher tool will automatically populate the coordinates for the last selection region
+  - If ```Configuration > General > Use last region``` is selected, Flameshot will always initialize with the last successfully captured region
+
+- Pinned screenshots can now be copied to the clipboard or saved to a file if a user right clicks on the pinned image by @zhangfuwen in https://github.com/flameshot-org/flameshot/pull/2519
+
+- Users can now specify their own Imgur API Key from ```Configuration > General > Imgur API Key```. This is encouraged because as Flameshot has gotten more popular we have started exceeding the upload limit of the default API key by@borgmanJeremy in https://github.com/flameshot-org/flameshot/pull/2503
+
+- Added 'Save to disk' button when uploading to imgur by @AndreaMarangoni in https://github.com/flameshot-org/flameshot/pull/2237
+
+- Pinned screenshots can now be zoomed with a pinch gesture by @AndreaMarangoni in https://github.com/flameshot-org/flameshot/pull/2447
+
+- The SVG's have been optimized by @RiedleroD in https://github.com/flameshot-org/flameshot/pull/2318
+
+- Make KDE use Freedesktop portal by @greenfoo in https://github.com/flameshot-org/flameshot/pull/2495
+
+- Allow final actions when printing geometry by @borgmanJeremy in https://github.com/flameshot-org/flameshot/pull/2444
+
+- Many Flameshot widgets have been reworked to use .ui XML files and Qt Designer. This has been done to allow non C++ developers to more easily contribute to the graphical side of Flameshot.
+<p align=center><img src="https://github.com/flameshot-org/flameshot/blob/master/docs/images/ui_file.png" width=50%> </p>
+
+- Updated Translations
+
```

Comment:
Was it in this version that right-clicking the tools would bring up its settings in the sidebar?

Reply:
Nope that seems to always work on v11 (I had to test to check lol)